mono/metadata/Makefile.am

```diff
@@ -10,11 +10,12 @@ win32_sources = \
 	marshal-windows-internals.h \
 	mono-security-windows.c \
 	mono-security-windows-internals.h \
-	process-windows.c \
-	process-windows-internals.h \
 	w32mutex-win32.c \
 	w32semaphore-win32.c \
 	w32event-win32.c \
+	w32process-win32.c \
+	w32process-win32-internals.h \
+	w32process-win32-uwp.c \
```

Comment:
We don't add uwp sources to the build configs yet since it is under development, so just leave it out from Makefile and vcxproj's for now.